native/NativeSampleApps/SmartSyncExplorer/src/com/salesforce/samples/smartsyncexplorer/ui/MainActivity.java

```diff
@@ -109,10 +108,7 @@
 	
     private SearchView searchView;
     private ContactListAdapter listAdapter;
-    private UserAccount curAccount;
 	private NameFieldFilter nameFilter;
-	private List<ContactObject> originalData;
-	private SmartStore smartStore;
```

Comment:
We don't interact with SmartStore in this class - ContactListLoader is the one that does that